.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+<!-- Before submitting your extensions, please *fully* read our contributing guidelines and our custom extension tutorial. Trust me, it is worthwhile and contains important information such as acceptance criteria and licensing.
+
+Pull requests that don't follow the guidelines will take *much* longer to be reviewed.
+
+Change [ ] to [x] to check off the box. -->
+
+- [ ] I have read the contributing guidelines
```

Comment:
```suggestion
- [ ] I have read the [contributing guidelines](https://github.com/TurboWarp/extensions/blob/master/CONTRIBUTING.md)
- [ ] I have read the [custom extension tutorial](https://docs.turbowarp.org/development/extensions/introduction) **in full**
```

Reply:
You can't click links from text fields while editing, unfortunately.

Reply:
I could probably add a note to the comment telling you where to go to find the contributing guidelines.